src/ks2e-m-dc5-element.ts

```diff
@@ -0,0 +1,308 @@
+          >
+            <path
+              transform="scale(.5)"
+              d="m5 0c0 2.76-2.24 5-5 5s-5-2.24-5-5 2.3-5 5-5c2.76 0 5 2.24 5 5z"
+              fill="context-stroke"
+              fill-rule="evenodd"
+              stroke="context-stroke"
+              stroke-width="1pt"
+            />
+          </marker>
+          <marker
+            id="Dot-7"
+            overflow="visible"
+            orient="auto"
+            preserveAspectRatio="xMidYMid"
+            viewBox="0 0 5.6666667 5.6666667"
+          >
+            <path
+              transform="scale(.5)"
+              d="m5 0c0 2.76-2.24 5-5 5s-5-2.24-5-5 2.3-5 5-5c2.76 0 5 2.24 5 5z"
+              fill="context-stroke"
+              fill-rule="evenodd"
+              stroke="context-stroke"
+              stroke-width="1pt"
+            />
+          </marker>
+          <marker
+            id="Dot-1-9-5"
+            overflow="visible"
+            orient="auto"
+            preserveAspectRatio="xMidYMid"
+            viewBox="0 0 5.6666667 5.6666667"
+          >
+            <path
+              transform="scale(.5)"
+              d="m5 0c0 2.76-2.24 5-5 5s-5-2.24-5-5 2.3-5 5-5c2.76 0 5 2.24 5 5z"
+              fill="context-stroke"
+              fill-rule="evenodd"
+              stroke="context-stroke"
+              stroke-width="1pt"
+            />
+          </marker>
+          <marker
+            id="Dot-9"
+            overflow="visible"
+            orient="auto"
+            preserveAspectRatio="xMidYMid"
+            viewBox="0 0 5.6666667 5.6666667"
+          >
+            <path
+              transform="scale(.5)"
+              d="m5 0c0 2.76-2.24 5-5 5s-5-2.24-5-5 2.3-5 5-5c2.76 0 5 2.24 5 5z"
+              fill="context-stroke"
+              fill-rule="evenodd"
+              stroke="context-stroke"
+              stroke-width="1pt"
+            />
+          </marker>
+          <marker
+            id="Dot-1-8"
+            overflow="visible"
+            orient="auto"
+            preserveAspectRatio="xMidYMid"
+            viewBox="0 0 5.6666667 5.6666667"
+          >
+            <path
+              transform="scale(.5)"
+              d="m5 0c0 2.76-2.24 5-5 5s-5-2.24-5-5 2.3-5 5-5c2.76 0 5 2.24 5 5z"
+              fill="context-stroke"
+              fill-rule="evenodd"
+              stroke="context-stroke"
+              stroke-width="1pt"
+            />
+          </marker>
+          <marker
+            id="Dot-7-5"
+            overflow="visible"
+            orient="auto"
+            preserveAspectRatio="xMidYMid"
+            viewBox="0 0 5.6666667 5.6666667"
+          >
+            <path
+              transform="scale(.5)"
+              d="m5 0c0 2.76-2.24 5-5 5s-5-2.24-5-5 2.3-5 5-5c2.76 0 5 2.24 5 5z"
+              fill="context-stroke"
+              fill-rule="evenodd"
+              stroke="context-stroke"
+              stroke-width="1pt"
+            />
+          </marker>
```

Comment:
What are all those markers? I don't think they are relevant to the drawing, so let's remove them